dependencies/jessie/apipie-params/copyright

```diff
@@ -0,0 +1,31 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: apipie-params
+Source: https://github.com/iNecas/apipie-params
+
+Files: *
+Copyright: 2012 Pavel Pkorny
+License: MIT
+
+Files: debian/*
+Copyright: 2013-2014 Red Hat Inc. <foreman-dev@googlegroups.com>
+License: MIT
+Comment: the Debian packaging is licensed under the same terms as the original package.
+
+License: MIT
+ Permission is hereby granted, free of charge, to any person obtaining a copy  
+ of this software and associated documentation files (the "Software"), to deal  
+ in the Software without restriction, including without limitation the rights  
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell  
+ copies of the Software, and to permit persons to whom the Software is  
+ furnished to do so, subject to the following conditions:  
+  
```

Comment:
dots are required for paragraphs here, see https://anonscm.debian.org/cgit/pkg-ruby-extras/ruby-chef-api.git/commit/?id=3d62fc5d6d81bb2296ee9d5daabc09dc94730c6c